src/seedu/addressbook/data/AddressBook.java

```diff
@@ -20,15 +22,20 @@
  */
 public class AddressBook {
 
-    private final UniquePersonList allPersons;
+    private static final String ACTION_ADD = "ADD";
+	private static final String ACTION_DELETE = "DELETE";
```

Comment:
as with other PRs, do note what is being entered. here this is a tab instead of four space for indentation